src/Nethermind/Nethermind.Core/Block.cs

```diff
@@ -155,5 +155,10 @@ public enum Format
             HashNumberDiffAndTx,
             Short
         }
+        
+        public virtual Block CreateBlockForProcessing(BlockHeader header) => 
```

Comment:
remove processing references from Nethermind.Core where possible, keep clean

Reply:
Ok rename here is good